### PR TITLE
added message of exception to stacktrace copy

### PR DESCRIPTION
**Pull request by sbke-mms**
# Description

Copying the stacktrace of an error in the Error Details view now also contains the message of the error.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by mbeuthan — APPROVED** [review]

File: report-ng/app/src/components/method-details/details.ts [hunk 1]
```diff
@@ -84,7 +84,10 @@ export class Details {
     }
 
     private _copyStackTraceToClipboard(stackTrace: IStackTraceCause[]) {
-        const msg = stackTrace.flatMap(cause => cause.stackTraceElements).join("\n");
+        let msg = "";
+        stackTrace.forEach(function (value) {
+            msg += value.message + "\n" + value.stackTraceElements.join("\n");
```
Comment (mreiche) [inline report-ng/app/src/components/method-details/details.ts:89]: Does this break the IntelliJ feature *Code* -> *Analyze Stack Trace*?
Bildschirmfoto 2023-01-06 um 16 45 51

**Review by mreiche — CHANGES_REQUESTED** [review]

File: report-ng/app/src/components/method-details/details.ts (hunk 1 above)
Comment (martingrossmann) [reply report-ng/app/src/components/method-details/details.ts:89]: No, it doesn't. IntellJ can handle this. We can also add some formations like indentations to make it more readable.

**Review by mreiche — DISMISSED** [review]

File: report-ng/app/src/components/method-details/details.ts [hunk 2]
```diff
@@ -84,7 +84,10 @@ export class Details {
     }
 
     private _copyStackTraceToClipboard(stackTrace: IStackTraceCause[]) {
-        const msg = stackTrace.flatMap(cause => cause.stackTraceElements).join("\n");
+        let msg = "";
```
Comment (martingrossmann) [inline report-ng/app/src/components/method-details/details.ts:87]: @sbke-mms 
The stacktrace can extended and also formated like this:
```java
const msg = stackTrace
            .flatMap(cause => cause.className + ": " + cause.message + "\n    " + cause.stackTraceElements.join("\n    "))
            .join("\n");
```

Comment (sbke-mms) [reply report-ng/app/src/components/method-details/details.ts]: @martingrossmann adjusted the message accordingly

**Review by martingrossmann — APPROVED** [review]

**Review by mreiche — APPROVED** [review]

**Pull request merged**